test/utils/setup.js

```diff
@@ -2,12 +2,6 @@ const formatUtil = require('format-util');
 const Mocha = require('mocha');
 const createDOM = require('./createDOM');
 
-process.browser = true;
-
-// Enable missing act warnings: https://github.com/facebook/react/blob/v16.13.1/packages/react-reconciler/src/ReactFiberHooks.js#L965
-// TODO: Revisit once https://github.com/facebook/react/issues/15439 is resolved.
-global.jest = null;
```

Comment:
better share it with karma

Reply:
Please leave this for another PR. It's brittle as is and almost nothing gained by it.

Reply:
i.e. this should not be part of this PR.

Reply:
Ok